### DISCUSSION TITLE
Choose a name for unit model costing block with alternatives for cost relationships

### DISCUSSION THREAD

**Discussion by adam-a-a** [watertap-org/watertap#1197] [Polls]
See PR #1183 which adds a "multiple choice costing block" so that the user could toggle between cost relationships for a given unit within the same flowsheet (i.e., without the need for iterating through discrete flowsheets simply to compare results with varying cost relationships).

The only thing that the PR is waiting on is the name we like best. I guess I started this when suggesting something other than `MultipleChoiceCostingBlock`. 

The nominees for best name are:
UnitModelCostingBlockChoices
MultiChoiceUnitModelCostingBlock
MultiChoiceCostingBlock ( unit model is implicit )
CostingBlockChoices ( unit model is implicit )
CostSelectorBlock

**Comment by adam-a-a** [comment]
I vote for `CostSelectorBlock`. This name aligns with the name of one of the methods under this class (`select_costing_block`), and it is the most concise of the bunch, as well as the clearest in terms of conveying its functionality (IMHO).

Definitely overthinking this, but thought it might be fun to try one of these polls out. =)

**Comment by TimBartholomew** [comment]
I think we should just tag "Multi" on whatever the name of the typical block is called (I think UnitModelCostingBlock), so MultiUnitModelCostingBlock.

Because that's in essence what it is, it allows someone to have multiple unit model costing blocks and toggle between them
